android/src/main/java/com/cloudwebrtc/webrtc/utils/AnyThreadResult.java

```diff
@@ -0,0 +1,32 @@
+package com.cloudwebrtc.webrtc.utils;
+
+import android.os.Looper;
+import android.os.Handler;
+
+import io.flutter.plugin.common.MethodChannel;
+
+public final class AnyThreadResult implements MethodChannel.Result {
+    final private MethodChannel.Result result;
```

Comment:
I think we can move the handler to a member variable so that we can reduce duplicate rows, if this doesn't throw any exceptions.
```final Handler handler = new Handler(Looper.getMainLooper());```

Reply:
Yeah, good idea, I will change that

Reply:
Maybe even static final?

Reply:
If it doesn't throw an exception, I think it's okay. :)

Reply:
I am not sure how the android thread works, assuming that different threads call the same handler.post at the same time.